Rakefile

```diff
@@ -20,14 +20,23 @@ Bundler::GemHelper.install_tasks
 
 require "rake/testtask"
 
-Rake::TestTask.new(:test) do |t|
-  t.libs = ["lib", "test"]
-  t.ruby_opts = ["-rsetup"]
-  t.pattern = "test/**/*_test.rb"
-  t.verbose = false
-
-  # Hide warnings emitted by our dependencies.
-  t.warning = false
+namespace :test do
+  ["postgresql", "mysql2"].each do |adapter|
+    adapter_short = adapter[/^[a-z]+/]
+    Rake::TestTask.new(adapter_short => "set_#{adapter_short}_env") do |t|
+      t.libs = ["lib", "test"]
+      t.ruby_opts = ["-rsetup"]
+      t.pattern = "test/**/*_test.rb"
+      t.verbose = true
```

Comment:
Seems the value's flipped from `false` in the original code to `true` here. Is that intentional?